### PR TITLE
Update database

### PR DESCRIPTION
**Pull request by phelian23**
In this PR
- Add a column to the table
- Update table with transactions
- Query table
- Add owners table
- Add species table
- Query tables using Join

### CODE REVIEW

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🎉🍾✔**
Hello @phelian23 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql [hunk 1]
```diff
@@ -3,4 +3,41 @@
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Agumon', '2020-02-03', 0, true, 10.23);
```
Comment (SajjadAhmad14) [inline data.sql:3]: [ OPTIONAL BUT RECOMMENDED ] Please always create a new feature branch for every new feature and create a new Pull request from that new branch.

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Femi, 
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!
## OPTIONAL
- [ ] Please see inline comments.
## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: data.sql (hunk 1 above)
Comment (phelian23) [reply data.sql:3]: Thanks...

**Pull request merged**